content/webapp/components/VideoTranscript/index.tsx

```diff
@@ -12,10 +12,12 @@ import { getFormatString } from '@weco/content/utils/iiif/v3';
 
 type Props = {
   supplementing: (ContentResource | ChoiceBody)[];
+  theme?: 'dark';
 };
 
 const VideoTranscript: FunctionComponent<Props> = ({
   supplementing,
+  theme,
```

Comment:
We've used the `isDark` boolean in the Audio Player and the ViewerButton, is that something we'd like to re-use when theming?

Reply:
Mostly because I thought this was passing the ThemeProvider theme in as I'm used to this  🙈 
<img width="292" alt="Screenshot 2025-05-07 at 13 33 26" src="https://github.com/user-attachments/assets/a5b2e2f5-1026-4151-a9a7-7b54e35532eb" />

Reply:
Good point, I'll change it